resources/packaging/macos/Info.plist

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>CFBundleDevelopmentRegion</key><string>English</string>
+        <key>CFBundleDisplayName</key><string>Sniffnet</string>
+        <key>CFBundleExecutable</key><string>wrapper.sh</string>
+        <key>CFBundleIconFile</key><string>sniffnet.icns</string>
+        <key>CFBundleIdentifier</key><string>io.github.gyulyvgc.sniffnet</string>
+        <key>CFBundleInfoDictionaryVersion</key><string>6.0</string>
+        <key>CFBundleName</key><string>Sniffnet</string>
+        <key>CFBundlePackageType</key><string>APPL</string>
+        <key>CFBundleShortVersionString</key><string>1.0.0</string>
```

Comment:
we should include sniffnet version from cargo manifest here

Reply:
I can adjust the CI workflow to do so on the fly...

Reply:
✅ Done!

Reply:
I've implemented this and adjusted the package file name as well.